client/src/components/Doctor/UpdateProfile.tsx

```diff
@@ -28,7 +28,7 @@ const UpdateProfile: FunctionComponent<{}> = () => {
     },
     {
       name: 'Sutter Health Care',
-      address: '0xB3CC507e752Dcc3DA1cEf955B58e97Ae77160103',
+      address: '0x29954F7DceDC3c5d7A44A3a69Abd101Acb62b98B',
```

Comment:
is this hardcoding necessary? HC provider registration already exists.

Reply:
Thank you for the review.
Yes, the code to fetch all the hcs from the blockchain is yet to be merged with main. 
https://codebrigadesjsu.atlassian.net/browse/HEAL-96 addresses this issue.

Reply:
can we pull these into separate file. if modification are needed, we can delete that reference and find out all files that are impacted. this will help us fix issues in future.